tests/integration/security/rbac/v2/basic/basic_test.go

```diff
@@ -80,72 +54,75 @@ func TestRBACV2(t *testing.T) {
 		// Port 80 is where HTTP is served, 90 is where TCP is served. When an HTTP request is at port
 		// 90, this means it is a TCP request. The test framework uses HTTP to mimic TCP calls in this case.
 		{Request: connection.Connection{To: appA, From: appB, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/xyz"},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
```

Comment:
Can we s/ExpectedRespCode/Expect`, s/DenyHTTPRespCode/HTTPCode403/ and s/AllowHTTPRespCode/HTTPCode200?

Then change the type of `Expect` to enum (available values are `None`, `Allow`, `Deny`), then you can just use the `HTTPCode403` and `HTTPCode200` in `CheckRBACRequest()`, this should significantly simplify the test case with better readability.

Reply:
I changed DenyHTTPRespCode to HTTPCode403 and AllowHTTPRespCode to HTTPCode200. The other change depends on the refactoring discussed here https://github.com/istio/istio/pull/13759#discussion_r280259737